src/pfe/file-watcher/server/src/controllers/projectStatusController.ts

```diff
@@ -78,8 +80,9 @@ class ProjectState {
      * @param appImageLastBuild <Optional | String> - The last app image info.
      * @param buildImageLastBuild <Optional | String> - The last build image info.
      * @param detailedAppStatus <Optional | DetailedAppStatus> - The detailed app status to update.
+     * @param notificationID <Optional | String> - Notification ID for IDE.
      */
-    constructor(state: any, msg: string, lastbuild?: number, appImageLastBuild?: string, buildImageLastBuild?: string, detailedAppStatus?: DetailedAppStatus) {
+    constructor(state: any, msg: string, lastbuild?: number, appImageLastBuild?: string, buildImageLastBuild?: string, detailedAppStatus?: DetailedAppStatus, notificationID?: string) {
```

Comment:
Based on the previous comment we should be able to remove `notificationID?: string` from the constructor as well.